egs/wsj/s5/steps/data/reverberate_data_dir.py

```diff
@@ -20,14 +20,14 @@ def GetArgs():
                                                  "--random-seed 1 data/train data/train_rvb",
                                      formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
-    parser.add_argument("--rir-list-file", type=str, required = True, 
+    parser.add_argument("--rir-list-file", type=str, action='append', required = True, dest = "rir_list_file_array", 
                         help="RIR information file, the format of the file is "
                         "--rir-id <string,required> --room-id <string,required> "
                         "--receiver-position-id <string,optional> --source-position-id <string,optional> "
                         "--rt-60 <float,optional> --drr <float, optional> location <rspecifier> "
                         "E.g. --rir-id 00001 --room-id 001 --receiver-position-id 001 --source-position-id 00001 "
                         "--rt60 0.58 --drr -4.885 data/impulses/Room001-00001.wav")
-    parser.add_argument("--noise-list-file", type=str, default = None,
+    parser.add_argument("--noise-list-file", type=str, action='append', default = None, dest = "noise_list_file_array",
```

Comment:
Once again I would recommend supporting assignment of different probability masses to different lists.